### PR TITLE
Istep updates - 07012020

### PR DESCRIPTION
**Pull request by ojayanth**
- IPL Flow v0.91.06 updates.
- updatehwmodel istep updates.

### CODE REVIEW

File: src/istep/p10/help/istep_list.htxt [hunk 1]
```diff
@@ -1,213 +1,214 @@
...
+host_activate_core                       16     Activate core
+host_activate_secondary_cores            16     Activate secondary cores
+host_secure_rng                          16     Secure the random number
+mss_scrub                                16     Start background scrub
+host_ipl_complete                        16     Notify SP drawer ipl complete
+
+collect_drawers                          17     Wait for all drawers to complete
+proc_psiinit                             17     PSI Initilization
+proc_diag                                17     PSI Diagnostics
+sys_proc_eff_config_links                18     Powerbus link config
```
Comment (lkammath) [inline src/istep/p10/help/istep_list.htxt:192]: Need leave a gap after step 17

File: src/istep/edbgIstep.C [hunk 2]
```diff
@@ -59,14 +59,14 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   { 0,   1,                                   "poweron",                   EDBG_ISTEP_NOOP},
```
Comment (lkammath) [inline src/istep/edbgIstep.C:59]: There is also a comment in the code mentioning version we need to update

File: src/istep/p10/help/istep_list.htxt (hunk 1 above)
Comment (ojayanth) [reply src/istep/p10/help/istep_list.htxt:192]: ACK

File: src/istep/edbgIstep.C (hunk 2 above)
Comment (ojayanth) [reply src/istep/edbgIstep.C:59]: updated in help file , any additioan change?

Comment (lkammath) [reply src/istep/edbgIstep.C:59]: in the code 
line#27 //Updated this table as per P10_IPL_Flow_v0.91.04.

Comment (ojayanth) [reply src/istep/edbgIstep.C:59]: Done

File: src/istep/edbgIstep.C [hunk 3]
```diff
@@ -214,12 +214,14 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   {15,   2,                        "proc_set_homer_bar",                   EDBG_ISTEP_HOST},
   {15,   3,                 "host_establish_ec_chiplet",                   EDBG_ISTEP_HOST},
   {15,   4,                    "host_start_stop_engine",                   EDBG_ISTEP_HOST},
-  {16,   1,                      "host_activate_master",                   EDBG_ISTEP_HOST},
-  {16,   2,                 "host_activate_slave_cores",                   EDBG_ISTEP_HOST},
```
Comment (lkammath) [inline src/istep/edbgIstep.C:218]: host_activate_boot_core

File: src/istep/p10/help/istep_list.htxt [hunk 4]
```diff
@@ -1,213 +1,215 @@
...
+proc_psiinit                             14     Enable PSI (For cronus only)
+proc_bmc_pciinit                         14     Enable PCIe for BMC (For cronus only)
+
+host_build_stop_image                    15     Build runtime STOP images
+proc_set_pba_homer_bar                   15     Set HOMER location in OCC
+host_establish_ex_chiplet                15     Select Hostboot core
+host_start_stop_engine                   15     Initialize the STOPGPE engine and
+                                                related functions to allow STOP to operate
+
+host_activate_core                       16     Activate core
```
Comment (lkammath) [inline src/istep/p10/help/istep_list.htxt:183]: host_activate_boot_core

Comment (ojayanth) [reply src/istep/p10/help/istep_list.htxt]: Done

**Pull request merged**